fe.py

```
@@ -20,11 +20,11 @@ def main(argv):
         LOG.info(now)
         dt = now.strftime("%Y %m %d %H %M")
         for sector in ["PR", "US", "AK", "HI", "GU"]:
-            cmd = f"sh production.sh {sector} {dt} A"
-            subprocess.call(cmd, shell=True)
+            cmd = ["bash", "production.sh", sector, dt, "A"]
+            subprocess.call(cmd)
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)